### PR TITLE
Add Icons for extension: "PT-BR".

### PR DESCRIPTION
**Pull request by nomaxsnx**
Checklist:

- [ ] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [ ] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [ ] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [ ] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [x] Have not changed source names
- [ ] Have explicitly kept the `id` if a source's name or language were changed
- [ ] Have tested the modifications by compiling and running the extension through Android Studio
- [x] Have removed `web_hi_res_512.png` when adding a new extension

### CODE REVIEW

File: src/pt/ladyestelarscan/res/mipmap-anydpi-v26/ic_launcher.xml
Comment (AwkwardPeak7) [inline src/pt/ladyestelarscan/res/mipmap-anydpi-v26/ic_launcher.xml:1]: remove

File: src/pt/ladyestelarscan/res/mipmap-hdpi/ic_launcher_background.png
Comment (AwkwardPeak7) [inline src/pt/ladyestelarscan/res/mipmap-hdpi/ic_launcher_background.png:1]: use the tool liked in contrubting guide to generate icons

**Review by AwkwardPeak7 — CHANGES_REQUESTED** [review]
all icons must follow the same convention, **sqaure shaped** with **rounded corners** and **solid background**

File: src/pt/ladyestelarscan/res/mipmap-anydpi-v26/ic_launcher.xml
Comment (nomaxsnx) [reply src/pt/ladyestelarscan/res/mipmap-anydpi-v26/ic_launcher.xml:1]: Done

File: src/pt/ladyestelarscan/res/mipmap-hdpi/ic_launcher_background.png
Comment (nomaxsnx) [reply src/pt/ladyestelarscan/res/mipmap-hdpi/ic_launcher_background.png:1]: Okay

**Comment by nomaxsnx** [comment]
> all icons must follow the same convention, **sqaure shaped** with **rounded corners** and **solid background**

Good afternoon, as I'm not very knowledgeable about "Colors", could you tell me which colors are good?

I'll send you a test on commit.

**Comment by AwkwardPeak7** [comment]
colour can be any that fit the background, the requirement is for a colour to be present, it can't be transparent.

new akimanga icon is good

**Comment by vetleledaal** [comment]
Make sure you add padding where it's warranted, currently these seem a bit small:
- flowermanga
- ghostscan
- hentaiteca


Others you touched seem fine:
- akimanga
- amuy
- astrumscans
- cafecomyaoi
- dessertscan
- kakuseiproject


And these are in the wrong shape, but you're actively working on it:
- ladyestelarscan
- leryaoi
- limboscan
- linkstartscan
- maidscan
- mangananquim
- sssscanlator

**Comment by nomaxsnx** [comment]
> Make sure you add padding where it's warranted, currently these seem a bit small:
> 
> * flowermanga
> * ghostscan
> * hentaiteca


10% Is it OK to apply to everyone?

image

 
image
obs: the logo looks a bit bad because of the text above, I don't know if you cut the text, just leave the ghost or not.

image

**Comment by vetleledaal** [comment]
Yes, 10% is good. Usually I try to keep the main subject within the grid lines.

For example:
image

**Review by AwkwardPeak7 — APPROVED** [review]

**Pull request merged**